moto/core/responses.py

```diff
@@ -482,14 +482,8 @@ def _get_multi_param_helper(
             tracked_prefixes or set()
         )  # prefixes which have already been processed
 
-        def is_tracked(name_param):
-            for prefix_loop in tracked_prefixes:
-                if name_param.startswith(prefix_loop):
-                    return True
-            return False
-
         for name, value in self.querystring.items():
-            if is_tracked(name) or not name.startswith(param_prefix):
```

Comment:
This was originally introduced for CloudWatch. SNS parameters are in a slightly different format (of course), so it could not be reused. It's not clear to me why this was introduced in the first place, but as far as I can tell, it doesn't do much - removing it ensures that both CW and SNS parameters are handled correctly